source/_posts/angular-signals.md

```diff
@@ -0,0 +1,409 @@
+---
+title: Angular Signals 🔥
+contributor: Łukasz Fajger
+avatar: lukasz-fajger.jpg
+description: A signal is a trackable value that changes over time. If the value of the signal changes, all related dependencies are automatically updated.
+tags: [angular, performance]
+id: angular-signals
+date: 2023-03-13
+---
+<!-- {% image_fw 1.78 angular-signals.png "Angular Signals" %}-->
+
+## Table of Contents
+<!-- toc -->
+
+{% image_fw 1.78 "banner.png" "Angular Signals" %}
+
+What if we could write even more performant applications in Angular without worrying about change detection? And what if we didn't want to render everything, but only the places that have exactly changed?
```

Comment:
but only the places that have exactly changed -> but only the **exact** places that have changed

Reply:
Fixed